aroquev/src/main/webapp/script.js

```diff
@@ -40,3 +40,36 @@ function calculateArmandoAge() {
     ageContainer.innerText = Math.abs(ageDt.getUTCFullYear() - 1970);
     
 }
+
+// Scripts for the image gallery
+var slideIndex = 1;
+
+function initSlides() {
+  showSlides(slideIndex);
+}
+
+// Next/previous controls
+function plusSlides(n) {
```

Comment:
Break this into 2 functions `showNextSlide()` and `showPrevSlide()` since there will only ever be  +/- 1 increments.

Reply:
Done